uber/config.py

```diff
@@ -13,15 +13,9 @@ def mixin(cls, klass):
     def include_plugin_config(self, plugin_config):
         """Plugins call this method to merge their own config into the global c object."""
 
-        for attr, val in plugin_config.items():
-            if not isinstance(val, dict):
-                setattr(self, attr.upper(), val)
-
-        if 'enums' in plugin_config:
-            self.make_enums(plugin_config['enums'])
-
-        if 'dates' in plugin_config:
-            self.make_dates(plugin_config['dates'])
```

Comment:
I don't think those other things are actually getting overridden in the `c` object - otherwise the app wouldn't run. I originally DID merge all the config files together through Sideboard, and it broke everything pretty bad due to things like `c.MODULE_ROOT` getting messed up. The `config_base` doesn't get re-parsed except for enums and dates, in other words.

Hm, on second thought, I think that may be leaving out top-level single config items.